main.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+// formats:
+// https://hostname/owner/repository.git
+// git@hostname:owner/repository.git
+// ssh://git@hostname:port/owner/repository.git
+func parseRepo(url string) (host string, owner string, name string) {
```

Comment:
`func parseRepo(url string) (host string, owner string,`**`name string)`** is not used